searx/engines/startpage.py

```diff
@@ -99,11 +99,16 @@ def response(resp):
         if re.match(r"^([1-9]|[1-2][0-9]|3[0-1]) [A-Z][a-z]{2} [0-9]{4} \.\.\. ", content):
             date_pos = content.find('...') + 4
             date_string = content[0:date_pos - 5]
-            published_date = parser.parse(date_string, dayfirst=True)
-
             # fix content string
             content = content[date_pos:]
 
+            # Startpage parser breaks on Spanish date format
```

Comment:
Please delete the comments. These lines are unnecessary.

Reply:
Done